lib/routers/rops.js

```diff
@@ -0,0 +1,47 @@
+const { Router } = require('express');
+
+module.exports = settings => {
+  const router = Router({ mergeParams: true });
+
+  router.get('/', (req, res, next) => {
+    const year = req.query.year || new Date().getFullYear();
```

Comment:
I would suggest making this throw if the year isn't provided. It will remove any risk of odd behaviour if default years are defined somewhere else differently.

Reply:
Good idea, will do.